rules/ml/ml_packetbeat_dns_tunneling.toml

```diff
@@ -28,6 +28,6 @@ references = ["https://www.elastic.co/guide/en/security/current/prebuilt-ml-jobs
 risk_score = 21
 rule_id = "91f02f01-969f-4167-8f66-07827ac3bdd9"
 severity = "low"
-tags = ["Elastic", "ML", "Packetbeat"]
+tags = ["Elastic", "Network", "Packetbeat", "ML"]
```

Comment:
I don't think `Packetbeat` fits into the tagging categorization scheme. I would eliminate that tag from all ML jobs that use it, because we don't do Auditbeat, and Winlogbeat tags.

Reply:
Removed all `Packetbeat` tags

Leaving this convo unresolved in case there are objections.